fbc-stage-index/release-resources/release-strategy.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleaseStrategy
+metadata:
+  name: e2e-fbc-stage-index-strategy
+  namespace: managed-release-team-tenant
+spec:
+  bundle: 'quay.io/hacbs-release/pipeline-fbc-release:main'
```

Comment:
I guess we use specific latest version number and not `main`, right @johnbieren ?

Reply:
That is what I'd suggest. If this moves into any e2e automation (I know for now they are manually executed), then we definitely shouldn't have main there. It can be fine for now though I guess